spec/defines/conf_spec.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 require 'spec_helper'
 
 describe 'logrotate::conf' do
-  _, facts = on_supported_os.first
-  let(:facts) { facts }
+  _os, os_facts = on_supported_os.first
```

Comment:
if `_os` is used, shouldn't it be named `os`?

Reply:
Done